assets/_scss/components/_header.scss

```diff
@@ -0,0 +1,374 @@
+header[role="banner"] {
```

Comment:
I'm not sure this is the best selector to encompass the header. One thing: do you know if this selector will work in other browsers? Besides that, as a developer, I wouldn't ever suspect a role attribute to effect styling. Another thing, this a strong selector (one lvl1 plus one lvl2). Is there a reason we shouldn't just make this a class?